tests/acceptance/features/bootstrap/SearchContext.php

```diff
@@ -62,8 +65,20 @@ public function userSearchesUsingWebDavAPI(
 		$body
 			= "<?xml version='1.0' encoding='utf-8' ?>\n" .
 			"	<oc:search-files xmlns:a='DAV:' xmlns:oc='http://owncloud.org/ns' >\n" .
-			"		<oc:search>\n" .
-			"			<oc:pattern>$pattern</oc:pattern>\n";
+			"		<oc:search>\n";
+		if ($scope !== null) {
+			$scope = \trim($scope, "/");
+			if ($this->featureContext->getDavPathVersion() === 3) {
+				$personal_spaceId = $this->featureContext->getPersonalSpaceIdForUser($user);
+				$scope = "$personal_spaceId/$scope";
+			} else {
+				$userId = $this->featureContext->getUserIdByUserName($user);
+				$scope = "$userId/$scope";
+			}
+			$body .= "<oc:pattern>*$pattern* scope:$scope</oc:pattern>\n";
+		} else {
+			$body .= "<oc:pattern>$pattern</oc:pattern>\n";
+		}
```

Comment:
more suggestion in my head :sweat_smile: 
```suggestion
		if ($scope !== null) {
			$scope = \trim($scope, "/");
			if ($this->featureContext->getDavPathVersion() === 3) {
				$rootPath = $this->featureContext->getPersonalSpaceIdForUser($user);
			} else {
				$rootPath = $this->featureContext->getUserIdByUserName($user);
			}
			$pattern .= " scope:$rootPath/$scope";
		}
		$body .= "<oc:pattern>$pattern</oc:pattern>\n";
```

Reply:
Please, refer to the suggested code change.